src/test/java/com/epam/tc/hw2/ex2/SecondExerciseTest.java

```diff
@@ -0,0 +1,52 @@
+package com.epam.tc.hw2.ex2;
+
+import static java.util.Map.entry;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.tc.hw2.BaseTest;
+import com.epam.tc.hw2.BaseUtil;
+import com.epam.tc.hw2.entities.User;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.Test;
+
+public class SecondExerciseTest extends BaseTest {
+
+    // path not to input node but to its parent which has text
+    public static final String CHECKBOX_ELEMENTS_XPATH = "//label[@class='label-checkbox']";
+    public static final String CHECKBOX_RADIO_XPATH = "//label[@class='label-radio']";
+    public static final String COLOR_XPATH = "//div[@class='colors']/*";
```

Comment:
Locators can't be a part of test class. It should be moved in Tests or inside methods.

Reply:
Moved to utility class